packages/common/src/dotlottie-player.ts

```diff
@@ -197,6 +205,8 @@ export class DotLottiePlayer {
 
   private _visibilityPercentage: number = 0;
 
+  private _howlerInstance: DotLottieAudio[] = [];
```

Comment:
can we call it `audios` instead of `_howlerInstance` ?

Reply:
@theashraf added